ted/src/main/java/frc/robot/commands/ShooterSpinUpForeverCommand.java

```diff
@@ -0,0 +1,69 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo - 2024
+// File: ShooterSpinUpForeverCommand.java
+// Intent: Forms a command to spin up the shooter outake motors 
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.control.Constants;
+import frc.robot.subsystems.ShooterOutfeedSubsystem;
+
+/**
+ * Forms a command to intake the note
+ * Intake is run until note is detected or timer has expired
+ */
+public class ShooterSpinUpForeverCommand extends Command
+{
+  private ShooterOutfeedSubsystem shooterOutfeed;
+  private boolean done = false;
+  
+  /** 
+  * Creates a new intake command 
+  * 
+  * @param shooterSubsystem - the shooter outfeed subsystem
+  */
+  public ShooterSpinUpForeverCommand(ShooterOutfeedSubsystem shooterSubsystem)
+  {
+    this.shooterOutfeed = shooterSubsystem;
+    addRequirements(shooterOutfeed);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize()
+  {
+    done = false;
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute()
+  {
+    if(!done){
+        shooterOutfeed.setShooterVelocityLeft(Constants.shooterLeftDefaultSpeedRpm);
+        shooterOutfeed.setShooterVelocityRight(Constants.shooterRightDefaultSpeedRpm);
+    }
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted)
+  {
+    if(interrupted)
```

Comment:
We are intentionally not setting the speeds to 0 here. We are assuming the next command will do something correct with the spun-up shooter. We should add a comment here stating that.